charts/datadog-crds/templates/datadoghq.com_datadogagents_v1.yaml

```diff
@@ -9412,10 +3802,4 @@ spec:
       storage: true
       subresources:
         status: {}
-status:
-  acceptedNames:
-    kind: ""
-    plural: ""
-  conditions: []
-  storedVersions: []
```

Comment:
i think this needs to be kept

Reply:
These got [removed](https://github.com/DataDog/datadog-operator/commit/ff6034791428a2c53240d614a9c33f2efda8ac79#diff-075817787997552bfd7680be6c03b4f787637fa1257b30f456c3d39b8b471884L193-L198) during controller-gen upgrade here https://github.com/DataDog/datadog-operator/pull/1247.